etc/avocado/conf.d/jobscripts.conf

```diff
@@ -0,0 +1,9 @@
+[avocado.plugins.jobscripts]
+# Directory with scripts to be executed before a job is run
+pre = /etc/avocado/scripts/job/pre.d/
+# Directory with scripts to be executed after a job is run
+post = /etc/avocado/scripts/job/post.d/
+# Warn if configured (or default) directory does not exist
+warn_non_existing_dir = False
+# Warn if any script run return non-zero status
+warn_non_zero_status = False
```

Comment:
I think better default would be to warn users about pre/post failures (the same way we warn them when plugin is not loaded).